### PR TITLE
Enable nullability in some PropertyGridView members

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Enable nullability in some PropertyGridView members.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/10038)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyGridView.cs [hunk 1]
```diff
@@ -3256,7 +3256,7 @@ protected override void OnMouseWheel(MouseEventArgs e)
             if (fullNotches != 0)
             {
                 int originalOffset = initialOffset;
-                int large = fullNotches * _scrollBar.LargeChange;
+                int large = fullNotches * _scrollBar!.LargeChange;
```
Comment (dreddy-work) [inline src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyGridView.cs:3259]: can you add debug assert  on  `_scrollBar` and see if we can avoid `!`?

File: src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyGridView.cs [hunk 2]
```diff
@@ -3463,9 +3463,9 @@ protected override void OnPaint(PaintEventArgs e)
         }
     }
 
-    private void OnGridEntryLabelDoubleClick(object s, EventArgs e)
+    private void OnGridEntryLabelDoubleClick(object? s, EventArgs e)
```
Comment (dreddy-work) [inline src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyGridView.cs:3466]: why define nullable? and mark non null immediately.  This is private method.

File: src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyGridView.cs [hunk 3]
```diff
@@ -3477,9 +3477,9 @@ private void OnGridEntryLabelDoubleClick(object s, EventArgs e)
         DoubleClickRow(row, gridEntry.Expandable, RowLabel);
     }
 
-    private void OnGridEntryValueDoubleClick(object s, EventArgs e)
+    private void OnGridEntryValueDoubleClick(object? s, EventArgs e)
```
Comment (dreddy-work) [inline src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyGridView.cs:3480]: same here.

File: src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyGridView.cs (hunk 2 above)
Comment (gpetrou) [reply src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyGridView.cs:3467]: This is used in the event handler.

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**